.github/workflows/deploy.yml

```diff
@@ -0,0 +1,28 @@
+# From https://dev.to/heroku/deploying-to-heroku-from-github-actions-29ej
+name: Push Container to Heroku
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  HEROKU_APP_NAME: tamuhack-registration
```

Comment:
Would it be better to use an environment variable for hackathon name, like ```${EVENT-NAME}-registration``` instead of having ```tamuhack-registration```?

Reply:
Ideally, yes it would. However, that would require defining `HEROKU_APP_NAME` in each step that requires Heroku, and would probably be more work for maintenance than just modifying `HEROKU_APP_NAME`.

Reply:
That was my worry, that makes sense.